twa

```diff
@@ -57,21 +57,20 @@ function fetch_respcode {
 }
 
 function get_header {
-  header="${1}"
+  header="${1,,}"
```

Comment:
That ends up spawning three processes: one for the subshell (`$()`), one for `echo`, and one for `tr`. It could be taken down to two via `$(tr '[:upper:] [:lower:]' <<< "${1}")`, but given that the `,,` operator has been in `bash` for a decade and we explicitly require `bash` 4 or above, I think we're okay here.

Reply:
Fair enough!